### ISSUE TITLE
Upgrade quorum

### ISSUE THREAD

**Issue by melnikaite** [synechron-finlabs/quorum-maker#117]
Currently Quorum Maker install Quorum version 2.2.1 (January 16 2019)

This version have disabled Constantinople version of EVM for dev network, so I can't test my contracts on this EVM

Are there any schedule with plan to upgrade Quorum?

**Comment by ab-chowdhury** [comment]
Yup, we will be sooner releasing a newer version of quorum maker with upgraded quorum version.
